### DISCUSSION TITLE
The steam JSON doesn't includes any category tag , any guidance how I can seperate these according to category

### DISCUSSION THREAD

**Discussion by d0tb0t71** [iptv-org/community#986] [Q&A]
The steam JSON doesn't includes any category tag , any guidance how I can seperate these according to category

**Comment by freearhey** [comment]
You just need to merge the data from `streams.json` and `channels.json` using the `channel` and `id` fields:

**streams.json**

```jsonc
  {
    "channel": "BBCOne.uk",
    "url": "http://1111296894.rsc.cdn77.org/LS-ATL-54548-6/index.m3u8",
    //...
  }
```

**channels.json**

```jsonc
  {
    "id": "BBCOne.uk",
    "categories": ["general"],
    //...
  }
```
